incubator/hnc/pkg/validators/object_test.go

```diff
@@ -0,0 +1,64 @@
+package validators
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+	"github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/forest"
+)
+
+func TestObject(t *testing.T) {
+	f := forest.NewForest()
+	o := &Object{Forest: f}
+	l := zap.Logger(false)
+
+	tests := []struct {
+		name     string
+		oldLabel string
+		oldValue string
+		newLabel string
+		newValue string
+		fail     bool
+	}{
+		{name: "LabelInheritedFrom label doesn't exist",
```

Comment:
I'd say something like "regular labels can be changed"

Reply:
Done.